src/plugins/move/MoveAction.js

```diff
@@ -63,14 +63,14 @@ export default class MoveAction {
     async onSave(changes) {
         this.startTransaction();
 
-        let inNavigationPath = this.inNavigationPath(this.object);
+        const inNavigationPath = this.inNavigationPath(this.object);
+        const parentDomainObjectpath = changes.location || [this.parent];
+        const parent = parentDomainObjectpath[0];
+
         if (inNavigationPath && this.openmct.editor.isEditing()) {
             this.openmct.editor.save();
```

Comment:
Something about this logic doesn't seem right. We really leaving edit mode by triggering the move action?

Reply:
Yeah that's been in there for a while, I'll test it out, if this is actually the case, which I think it is, is there another solution you'd like me to implement?

Reply:
Yep, just straight up saves whatever you had edited up to the point and moves it. I can add a confirmations dialog... I feel like that makes sense.

Reply:
Ok, removed for MoveAction, left it in RemoveAction as that's where it was originally introduced.